pkg/asset/machines/openstack/machines.go

```diff
@@ -78,9 +68,8 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 			failureDomain,
 		)
 		if err != nil {
-			return nil, err
+			return nil, nil, fmt.Errorf("failed to generate the Machine providerSpec for replica %d: %w", idx, err)
```

Comment:
```suggestion
			return nil, nil, errors.Wrapf(err, "failed to generate the Machine providerSpec for replica %d", idx)
```

Reply:
Dave Cheney's `pkg/errors` [has been archived](https://github.com/pkg/errors) and all its functionality has been incorporated into the standard library. I know the dependency is still available in the Installer's `go.mod`, however I prefer not to use it in new code.